packages/eslint-config-ndla/index.mjs

```diff
@@ -0,0 +1,258 @@
+      Boolean,
+    ),
+    files: ["**/*.{jsx,tsx,mtsx,ctsx}"],
+    plugins: {
+      "react-hooks": hooksPlugin,
+    },
+    rules: {
+      "react/display-name": "off",
+      "react/prop-types": "off",
+      "react/no-unused-state": "warn",
+      "react/button-has-type": "error",
+      "react/jsx-pascal-case": "warn",
+      "react/style-prop-object": "warn",
+      "react/jsx-uses-react": "error",
+      "react/no-unknown-property": "error",
+      "react/forward-ref-uses-ref": "error",
+      "react/jsx-no-leaked-render": "error",
+      "react/jsx-no-useless-fragment": "error",
+      "react-hooks/rules-of-hooks": "error",
+      "react-hooks/exhaustive-deps": "warn",
+    },
+  },
+  jsxA11y.flatConfigs.recommended,
+  {
+    files: ["**/*.{jsx,tsx,mtsx,ctsx}"],
+    plugins: {
+      import: importPlugin,
+    },
+    rules: {
+      "import/first": "error",
+      "import/no-amd": "error",
+      // TODO: There's a perf issue with this rule. It's disabled until it's fixed.
+      // "import/no-cycle": ["warn", { maxDepth: Infinity }],
+      "import/no-anonymous-default-export": "error",
+      // TODO: Replace this with the simple-import-sort plugin
+      "import/order": [
+        "warn",
+        {
+          alphabetize: {
+            order: "asc",
+            caseInsensitive: true,
+          },
+          groups: ["builtin", "external", "internal"],
+          pathGroupsExcludedImportTypes: [],
+          pathGroups: [
+            {
+              pattern: "@ndla/**",
+              group: "internal",
+              position: "before",
+              patternOptions: { matchBase: true },
+            },
+            {
+              pattern: "@*/**",
+              group: "external",
+              position: "after",
+              patternOptions: { matchBase: true },
+            },
+          ],
+        },
+      ],
+    },
+  },
+  {
+    plugins: {
+      header: pluginHeader,
+    },
+    rules: {
+      "header/header": [
+        2,
+        "block",
+        [
+          "*",
+          {
+            pattern: " * Copyright \\(c\\) \\d{4}-present, NDLA.",
+            template: ` * Copyright (c) ${new Date().getFullYear()}-present, NDLA.`,
+          },
+          " *",
+          " * This source code is licensed under the GPLv3 license found in the",
+          " * LICENSE file in the root directory of this source tree.",
+          " *",
+          " ",
+        ],
+      ],
+    },
+  },
+  {
+    files: ["**/*.stories.{js,mjs,cjs,ts,jsx,tsx,mts,cts,mtsx,ctsx}"],
+    rules: {
+      "react/no-unescaped-entities": "off",
+    },
+  },
+  // {
+  //   plugins: {
+  //     "simple-import-sort": simpleImportSort,
+  //   },
+  //   rules: {
+  //     "simple-import-sort/imports": [
+  //       "error",
+  //       {
+  //         groups: [["^\\u0000", "^node:", "^\\w", "^@?\\w", "^", "^\\."]],
+  //       },
+  //     ],
+  //   },
+  // },
+);
```

Comment:
Er dette den andre pluginen du tenkte på? Er det ikkje berre å ta inn dersom den er kjappere?

Reply:
Nei, for den sorterer på en annen måte enn vi gjør nå. Tenker det er like greit å ta en migrering som kun fokuserer på dette senere.

Reply:
ok